content/posts/macvdmtool.md

```diff
@@ -3,51 +3,56 @@ title: "DFU restore Apple Silicon Macs with macvdmtool"
 date: 2021-11-04
 draft: false
 ---
+(updated 2024-01-31, new info about Intel macs)
 
 # What is [macvdmtool](https://github.com/AsahiLinux/macvdmtool)?
 
-It is a tool written by [marcan](https://github.com/marcan) under the [AsahiLinux project](https://asahilinux.org/). macvdmtool lets you put an Apple Silicon Mac in DFU mode with terminal commands instead of requiring awkward keyboard gymnastics.
+It is a tool written by [marcan](https://github.com/marcan) under the [AsahiLinux project](https://asahilinux.org/). macvdmtool lets you put any Apple Silicon or Intel Macs with the T2 chip (2018 and forward) in DFU mode with terminal commands instead of requiring awkward keyboard gymnastics.
 
-> **Note**: macvdmtool requires that you have _two_ Apple silicon machines on hand. One acts as the "host" machine which initiates the "target" machine to perform a restore. 
+> **Note**: macvdmtool requires that the **host machine** is **Apple Silicon** based, Intel macs do not seem to support this tool as a host.
 
 # Setting up and using macvdmtool
 
-1. Install Xcode and the Xcode commandline tools.
+1. Install [Xcode](https://developer.apple.com/xcode/) and the [Xcode commandline tools](https://mac.install.guide/commandlinetools/).
```

Comment:
Is there a more official link we can use for Xcode Commandline tools? Or just suggest the terminal command to install them?

Reply:
I am not sure, i have only found this: https://forums.developer.apple.com/forums/thread/677124

Reply:
I'd prefer to just provide the terminal commands directly instead of linking here

Reply:
Added terminal commands